pkg/tarfs/tarfs.go

```diff
@@ -189,10 +189,14 @@ AddEnt:
 		switch parent.h.Typeflag {
 		case tar.TypeDir:
 			// OK
+		case tar.TypeLink:
+			// This is annoying -- hard linking to directories is weird
+			fallthrough
 		case tar.TypeSymlink:
 			dir = parent.h.Linkname
 			goto AddEnt
 		default:
+			println(fmt.Sprintf("???: %s\t%s\t%#+v", name, dir, parent.h))
```

Comment:
Do we want to nix this line or convert to a log line?

Reply:
oops